plugins/headless/docs/headless-gamma-workflow.md

```diff
@@ -35,10 +35,6 @@ After exporting, the webserver has to be started. The server can be found in `hu
 
 Alternatively, the server can be started from terminal using Maven commands. Navigate to the folder containing the `pom.xml` file and enter the `mvn package` command. After the process finishes, enter `mvn exec:java`, and the server should start.
 
-Before running the server, make sure to configure the paths found in `config.properties`.
-- Set `headless.gamma.path` variable to the location of the exported `eclipse.exe` (or equivalent binary on Linux systems), and
```

Comment:
Keep these lines and add in parentheses that the user does not have to do anything with this if they have followed the instructions on "Exporting the products" (that is, they used the correct paths when exporting).